src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java

```diff
@@ -257,8 +257,56 @@ public void nellaSectionAllegatiCliccareSulBottoneInvia() {
 
         AllegatiPASection allegatiPASection = new AllegatiPASection(this.driver);
         allegatiPASection.selectInviaButton();
+        if (allegatiPASection.verificaMessaggioErrore()){
+            aggiornamentoNumeroProtocolloAllegati();
+            logger.error("Si vede il messaggio di dati non corretti");
+            Assert.fail("Si vede il messaggio di dati non corretti");
+        }
     }
+    private void aggiornamentoNumeroProtocolloAllegati() {
+        logger.info("Aggiornamento del numero protocollo");
+
+        DataPopulation dataPopulation = new DataPopulation();
+        String numeroProtocolOld = dataPopulation.readDataPopulation("datiNotifica.yaml").get("numeroProtocollo").toString();
```

Comment:
insert string in a variable, path and 'numeroProtocollo'

Reply:
done